en/airframes_experimental/traxxas_stampede.md

```diff
@@ -0,0 +1,122 @@
+# Traxxas Stampede VXL
+
+{% youtube %}https://youtu.be/N3HvSKS3nCw{% endyoutube %}
+
+![Traxxas Stampede VXL](../../assets/airframes/experimental/stampede/stampede.jpg)
+
+## Parts List
+
+  * [Traxxas Stampede](https://traxxas.com/products/models/electric/stampede-vxl-tsm) All of this is used except for the top plastic cover.
+  * [Pixhawk Mini](https://store.3dr.com/products/3dr-pixhawk)
+  * [Power Module](https://store.3dr.com/products/10s-power-module)
+  * [Telemetry Module (EU)](https://store.3dr.com/products/433-mhz-telemetry-radio) US version available in the website
+  * [Spektrum Dxe Controller](http://www.spektrumrc.com/Products/Default.aspx?ProdId=SPM1000) or other PX4-compatible remotes
+  * [Spektrum Quad Race Serial Receiver w/Diversity](http://www.spektrumrc.com/Products/Default.aspx?ProdID=SPM4648)
+  * [PX4Flow](https://pixhawk.org/modules/px4flow)
+
+
+## Assembly
+
+The assembly consists of a wooden frame on which all the parts were attached. Tests showed that a better vibration insulation should be used, especially for the Pixhawk and the Flow module.
```

Comment:
Here I meant better than just using cable ties on wood. This comes from an email I got where I was asked what I would improve if I were to improve something and I believe this can be seen here:

[http://logs.uaventure.com/view/ZuwDbp3UWHcf2KEVnxEBNb#IMU_Acc_PLOT](http://logs.uaventure.com/view/ZuwDbp3UWHcf2KEVnxEBNb#IMU_Acc_PLOT
)

This is a log I did on the grass and I noticed that quadrotors generally have raw accelerations one order of magnitude smaller, here I just pulled a log from someone in the community flying a F550 frame for comparison:
[http://logs.uaventure.com/view/ZuwDbp3UWHcf2KEVnxEBNb#IMU_Acc_PLOT](http://logs.uaventure.com/view/ZuwDbp3UWHcf2KEVnxEBNb#IMU_Acc_PLOT)